runtime/hydradx/src/assets.rs

```diff
@@ -1056,6 +1063,7 @@ impl AmmTradeWeights<Trade<AssetId>> for RouterWeightInfo {
 				}
 				PoolType::XYK => weights::pallet_xyk::HydraWeight::<Runtime>::router_execution_buy(c, e)
 					.saturating_add(<Runtime as pallet_xyk::Config>::AMMHandler::on_trade_weight()),
+				PoolType::Aave => Weight::zero(),
```

Comment:
any specific reason that the weight is zero? Same for `calculate_spot_price_with_fee_weight`

Reply:
cos i just return constant

Reply:
1